test_runner/fixtures/pageserver/http.py

```diff
@@ -210,16 +210,25 @@ def tenant_list(self) -> List[Dict[Any, Any]]:
         return res_json
 
     def tenant_create(
-        self, new_tenant_id: TenantId, conf: Optional[Dict[str, Any]] = None
+        self,
+        new_tenant_id: TenantId,
+        conf: None | Dict[str, Any] = None,
```

Comment:
Why did you change this to `None |` instead of `Optional[Dict[..]]`? I don't know, I am not familiar with python enough to know the answer.

Reply:
Hmm, I imagine that I was working around some issue with the linter when I wrote this, but when I try reverting this to Optional the linter is also happy.  

Switched back to optional in f0eda5429